synapse/__init__.py

```diff
@@ -21,6 +21,7 @@
 # the built-in cortex modules...
 BASE_MODULES = (
     ('synapse.models.syn.SynMod', {}),
+    ('synapse.models.axon.AxonMod', {}),
```

Comment:
OK, I'll add it back into there. Is it OK to leave the `synapse/models/axon.py` file? It feels a lot cleaner than defining it all within the axon code.